repository/n.json

```diff
@@ -914,6 +914,17 @@
 				}
 			]
 		},
+		{
+			"name": "Notepad++ Tab Context",
+			"details": "https://github.com/josephmorrill/SublimeNppTabContextPlugin",
+			"labels": ["tab context", "notepad++"],
+			"releases": [
+				{
+					"sublime_text": ">3211",
```

Comment:
This prevents installation on 3211, the latest available build. Is this what you want?

The last ST3 dev build is 3210 and would be exempted by this selector as well, despite being almost equivalent.

Reply:
Thanks! Updated the filter to >=3000